types.go

```diff
@@ -2,8 +2,51 @@ package venom
 
 import (
 	"encoding/xml"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+const (
+	// DetailsLow prints only summary results
+	DetailsLow = "low"
+	// DetailsMedium prints progress bar and summary
+	DetailsMedium = "medium"
+	// DetailsHigh prints progress bar and details
+	DetailsHigh = "high"
 )
 
+type (
```

Comment:
if you decide to make a `type block`, you should put every in it.
it's up to you, but only one `type block` of repeat `type` everywhere